src/script/standard.h

```diff
@@ -90,6 +88,7 @@ struct ScriptHash : public BaseHash<uint160>
     explicit ScriptHash(const CScript& script);
     explicit ScriptHash(const CScriptID& script);
 };
+CScriptID ToScriptID(const ScriptHash& script_hash);
```

Comment:
What would be const? Returning a const object (rather than a const reference/pointer) is [largely meaningless](https://stackoverflow.com/questions/8716330/purpose-of-returning-by-const-value).